source/html/renderer/src/host-config.ts

```diff
@@ -17,46 +17,56 @@ function parseSpacingDefinition(obj: any): ISpacingDefinition {
     } : null;
 }
 
-export interface IPaddingDefinition {
-    top: Enums.Padding,
-    right: Enums.Padding,
-    bottom: Enums.Padding,
-    left: Enums.Padding
+export class PaddingDefinition {
+    top: Enums.Padding;
+    right: Enums.Padding;
+    bottom: Enums.Padding;
+    left: Enums.Padding;
+
+    toJSON() {
+        return {
+            top: Enums.Padding[this.top],
+            right: Enums.Padding[this.right],
+            bottom: Enums.Padding[this.bottom],
+            left: Enums.Padding[this.left],
+        }
+    }
 }
 
-function parsePaddingDefinition(obj: any): IPaddingDefinition {
-    return obj ? {
+function parsePaddingDefinition(obj: any): PaddingDefinition {
```

Comment:
Since PaddingDefinition is now a class, this method should be moved to the class and should be renamed into simply "parse"